eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/ImpTest.hs

```diff
@@ -913,6 +915,15 @@ trySubmitTx tx = do
           | otherwise = error "Root not found in UTxO"
     impRootTxInL .= newRoot
     pure txFixed
+  where
+    fixedUp = do
+      doFixup <- asks iteDoTxFixup
+      let fx =
+            if doFixup
+              then fixupTx tx
+              else pure tx
+      postFixup <- asks itePostFixup
+      fx >>= postFixup
```

Comment:
Ok, I was thinking maybe it can make the main thing easier to read, by reducing all the fixing up to one invocation, but on the other hand of course, it makes you have to jump to see what's going on. 
Happy to remove the binding, I don't have any strong opinion on it :+1: